contracts/settling_game/L01_Settling.cairo

```diff
@@ -63,6 +65,26 @@ func settle{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(t
     # TODO: TimeStamp - current Hardcoded
     IS01_Settling.set_time_staked(settle_state_address, token_id, block_timestamp)
 
+    # updated settled realms counter
+    let (realms_settled) = IS01_Settling.get_total_realms_settled(contract_address=settle_state_address)
+    
+    IS01_Settling.set_total_realms_settled(settle_state_address, realms_settled + 1)
+
+    # update wonders
+    let (realms_data : RealmData) = realms_IERC721.fetch_realm_data(
+        contract_address=realms_address, token_id=token_id)
+
+    if realms_data.wonder == 1:
+        let (wonders_state_address) = IModuleController.get_module_address(
+        contract_address=controller, module_id=9)
```

Comment:
Why `module_id=9`? Isn't Wonders' ID 5? If so, applies to code below as well.

Reply:
L1: id=1
S1: id=2
L2: id=3
S2: id=4
L3: id=5
S4: id=6
L4: id=7
S4: id=8
L5: id=9
S5: id=10
?

Reply:
Yes its just the offset for the wonder state address in the controller's storage